README.md

```diff
@@ -53,6 +53,13 @@ The extension follows Kedro [pipeline autodiscovery mechanism](https://docs.kedr
 │           └── second_pipeline
 
 ```
+
```

Comment:
```suggestion
To visualize your Kedro project using Kedro-Viz in Visual Studio Code, follow these steps:

**Open the Command Palette**: 
Press Cmd + Shift + P (on macOS) or Ctrl + Shift + P (on Windows/Linux).

**Run Kedro-Viz**:
Type `kedro: Run Kedro Viz` and select the command.

This will launch Kedro-Viz and display your pipeline visually within the extension.
```